src/components/MoreClients.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Web from '../../static/img/web.svg';
+import Desktop from '../../static/img/monitor.svg';
+import { Android, Apple, Roku, Amazon, Kodi } from '@icons-pack/react-simple-icons';
```

Comment:
is there a chromecast icon?

Reply:
There is, but that's not really a client per se (As in you can't run it by itself, it needs another client).

I see it more as a feature than as a client, hence it not being included here.